src/logic/contracts/methodIds.ts

```diff
@@ -80,10 +80,84 @@ export const decodeParamsFromSafeMethod = (data: string): DataDecoded | null =>
   }
 }
 
+export const SPENDING_LIMIT_METHODS_NAMES = {
+  ADD_DELEGATE: 'addDelegate',
+  SET_ALLOWANCE: 'setAllowance',
+  EXECUTE_ALLOWANCE_TRANSFER: 'executeAllowanceTransfer',
+}
+
+export const SPENDING_LIMIT_METHOD_TO_ID = {
```

Comment:
this seems like id to a method, because the keys are IDs

Reply:
yes, that messed with my head too. That means that the `METHOD_TO_ID` const for Safe's methods, should be renamed as well. Isn't it?

Reply:
refactored a bit, moved these objects to the type file and renamed